local.go

```diff
@@ -31,7 +45,7 @@ type LocalTest struct {
 	// All single nodes
 	Nodes []*TreeNodeInstance
 	// Don't test for leaky goroutines
```

Comment:
Can you fix this comment to be: `// How carefully to check for leaking resources at the end of the test.`